nbs/Agents/01_AStrategyActorCritic.ipynb

```diff
@@ -100,7 +100,7 @@
     "\n",
```

Comment:
<p>This file has effectively no changes. It should be removed from the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wbarfuss/pyCRLD/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I'm testing ReviewNB. Looks nice.